analyzers/src/SonarAnalyzer.VisualBasic/Trackers/VisualBasicConstantValueFinder.cs

```diff
@@ -36,5 +37,9 @@ public class VisualBasicConstantValueFinder : ConstantValueFinder<IdentifierName
 
         protected override VariableDeclaratorSyntax VariableDeclarator(SyntaxNode node) =>
             node?.Parent as VariableDeclaratorSyntax;
+
+        protected override bool IsPtrZero(SyntaxNode node) =>
```

Comment:
is there a test for this?

Please add a test, I've seen previously code for `vb.net` w/o code coverage which proved to be faulty

why is this covered w/o a test?....

Reply:
The way I see it, there are 2 branches:
- when node is not `MemberAccessExpressionSyntax`: this is tested when analyzing a node as `-1`, see for ex: https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.vb#L19
- when node is `MemberAccessExpressionSyntax`: this is tested when analyzing a node as `Types.Class`, see for ex: https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SillyBitwiseOperation.vb#L49

Both of them are covered.

The last line `&& memberAccess.IsPtrZero(SemanticModel)` is not really a branch since the result is returned directly.